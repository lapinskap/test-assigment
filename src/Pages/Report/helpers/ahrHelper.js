export const getAhrUrl = (url, isAhr) => {
  let result = isAhr ? '/ahr' : '';
  result += url;

  return result;
};

export const asd = () => {};
