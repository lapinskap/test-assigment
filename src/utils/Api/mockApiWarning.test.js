import React from 'react';
import { act, create } from 'react-test-renderer';
import ApiWarning from './mockApiWarning';
import { waitToLoadMocks } from '../../tests/helpers';

function createNodeMock() {
  return {
    getBoundingClientRect() {
      return {
        width: 100,
      };
    },
  };
}

describe('<ApiWarning />', () => {
  afterEach(() => {
    window.SKIP_TEST_MODE = false;
    jest.clearAllMocks();
  });

  it('Should render correct with isMockView true', async () => {
    let tree = null;
    await act(async () => {
      tree = create(
        <ApiWarning />, { createNodeMock },
      );
    });
    await waitToLoadMocks();

    expect(tree.toJSON()).toMatchSnapshot();
  });

  it('Should render with isMockView false', async () => {
    let tree = null;
    window.SKIP_TEST_MODE = true;
    await act(async () => {
      tree = create(
        <ApiWarning />, { createNodeMock },
      );
    });
    await waitToLoadMocks();

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
