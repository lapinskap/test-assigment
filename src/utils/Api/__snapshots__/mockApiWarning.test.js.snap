// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApiWarning /> Should render correct with isMockView true 1`] = `
<div
  className="sticky-outer-wrapper"
  style={Object {}}
>
  <div
    className="sticky-inner-wrapper"
    style={
      Object {
        "position": "relative",
        "top": "",
        "transform": "translate3d(0,0px,0)",
        "zIndex": "999",
      }
    }
  >
    <div
      className="mb-0 text-center text-uppercase alert alert-warning fade"
      role="alert"
    >
      Wyświetlane dane są danymi testowymi
    </div>
  </div>
</div>
`;

exports[`<ApiWarning /> Should render with isMockView false 1`] = `null`;
